### PR TITLE
 Add a pointer to the IG for error code 69

### PR DESCRIPTION
**Pull request by omnivion**
### Information
- JIRA story QPPCT-844.

### Checklist 
- [ ] All JUnit tests pass (`mvn clean verify`).
- [x] New unit tests written to cover new functionality.
- [x] Added and updated JavaDocs for non-test classes and methods.
- [x] No local design debt. Do you feel that something is "ugly" after your changes?
- [x] Updated documentation (`README.md`, etc.) depending if the changes require it.

### CODE REVIEW

**Comment by FlexionSonarQube** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [ErrorCode.java#L72](https://github.com/CMSgov/qpp-conversion-tool/blob/a1f47ca54b231895c832b6413c9a0a91587ade97/commons/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java#L72): Define a constant instead of duplicating this literal "Please see the Implementation Guide for information on the performance period here: " 3 times. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'

**Comment by FlexionSonarQube** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [ErrorCode.java#L72](https://github.com/CMSgov/qpp-conversion-tool/blob/a0321e756aff832f16159ba9a06ffbe18614e831/commons/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java#L72): Define a constant instead of duplicating this literal "Please see the Implementation Guide for information on the performance period here: " 3 times. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'

File: commons/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java [hunk 1]
```diff
@@ -66,13 +66,13 @@
 	CLINICAL_DOCUMENT_CONTAINS_DUPLICATE_ECQM_SECTIONS(28, "Clinical Document contains duplicate Measure "
 			+ "sections"),
 	REPORTING_PARAMETERS_MUST_CONTAIN_SINGLE_PERFORMANCE_START(29, "Must have one and only one performance period "
-			+ "start. Please see the Implementation Guide for information on the performance period here: "
+			+ "start. Please see the Implementation Guide for information on the performance period here: " //NOSONAR
```
Comment (FlexionSonarQube) [inline commons/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java]: MAJOR Is //NOSONAR used to exclude false-positive or to hide real quality flaw ? [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ANoSonar)

File: commons/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java [hunk 2]
```diff
@@ -66,13 +66,13 @@
...
 			+ DocumentationReference.PERFORMANCE_PERIOD),
 	REPORTING_PARAMETERS_MUST_CONTAIN_SINGLE_PERFORMANCE_END(30, "Must have one and only one performance period end. "
-			+ "Please see the Implementation Guide for information on the performance period here: "
+			+ "Please see the Implementation Guide for information on the performance period here: " //NOSONAR
```
Comment (FlexionSonarQube) [inline commons/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java]: MAJOR Is //NOSONAR used to exclude false-positive or to hide real quality flaw ? [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ANoSonar)

File: commons/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java [hunk 3]
```diff
@@ -66,13 +66,13 @@
...
 			+ DocumentationReference.PERFORMANCE_PERIOD),
 	REPORTING_PARAMETERS_MISSING_PERFORMANCE_YEAR(31, "Must have a performance year. "
-			+ "Please see the Implementation Guide for information on the performance period here: "
+			+ "Please see the Implementation Guide for information on the performance period here: " //NOSONAR
```
Comment (FlexionSonarQube) [inline commons/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java]: MAJOR Is //NOSONAR used to exclude false-positive or to hide real quality flaw ? [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ANoSonar)

**Comment by FlexionSonarQube** [comment]
SonarQube analysis reported 3 issues
* ![MAJOR][MAJOR] 3 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: commons/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java (hunk 3 above)
Comment (omnivion) [reply commons/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java]: FALSE POSITIVE

File: commons/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java (hunk 2 above)
Comment (omnivion) [reply commons/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java]: FALSE POSITIVE

File: commons/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java (hunk 1 above)
Comment (omnivion) [reply commons/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java]: FALSE POSITIVE

**Pull request merged**